media-source-respec.html

```diff
@@ -2706,6 +2714,367 @@ <h3>Event Summary</h3>
+                  The user agent does not support |type|'s [=MIME type/subtype=].
+                </li>
+                <li>
+                  If |type| has [=MIME type/parameters=], if any of the codecs
+                  passed as [=MIME type/parameters=] is not supported.
+                </li>
+              </ul>
+            </li>
+            <li>Otherwise, return `true`.</li>
+          </ol>
+          <p class="note">
+            If true is returned from this method, it only indicates that the
+            {{ManagedMediaSource}} implementation is capable of creating
+            {{ManagedSourceBuffer}} objects for the specified MIME type. An
+            {{MediaSource/addSourceBuffer()}} call can still fail if there are
+            insufficient resources available to support the addition of a new
+            {{ManagedSourceBuffer}}.
+          </p>
+          <p class="note">
+            This method returning true implies that {{HTMLMediaElement}}'s
+            {{HTMLMediaElement/canPlayType()}} will return
+            {{CanPlayTypeResult/"maybe"}} or {{CanPlayTypeResult/"probably"}} since
+            it does not make sense for a {{ManagedMediaSource}} to support a type
+            the {{HTMLMediaElement}} knows it cannot play.
+          </p>
+        </dd>
+      </dl>
+
+      <h3>Event Summary</h3>
+      <table class="old-table">
+        <thead>
+          <tr>
+            <th>Event name</th>
+            <th>Interface</th>
+            <th>Dispatched when...</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <dfn class="event">startstreaming</dfn>
+            </td>
+            <td>{{Event}}</td>
+            <td>
+              {{ManagedMediaSource/streaming}} attribute changes from `false` to `true`.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <dfn class="event">endstreaming</dfn>
+            </td>
+            <td>{{Event}}</td>
+            <td>
+              {{ManagedMediaSource/streaming}} attribute changes from `true` to `false`.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3>Algorithms</h3>
+      <h4>`ManagedSourceBuffer` Monitoring</h4>
+      <p>
+        The following steps are run periodically, whenever the {{MediaSource}}
+        {{SourceBuffer}} Monitoring algorithm is scheduled to run.
+      </p>
+      <p>
+        Having <dfn>enough managed data to ensure uninterrupted playback</dfn>
+        is an implementation specific condition where the user agent determines that
+        it currently has enough data to play the presentation without stalling for a
+        meaningful period of time and that it can achieve a balance between optimal
+        memory and energy usage. This condition is constantly evaluated to determine
+        when to transition the media element into and out of the
+        {{HTMLMediaElement/HAVE_ENOUGH_DATA}} ready state. These transitions
```

Comment:
Changing HAVE_ENOUGH_DATA semantics seems a bit risky, but likely fine for a new API like this -- though I suspect many clients will just blindly change MSE to MMSE. I was assuming all new state changes would come through the new events and readyState would be unchanged relative to normal MSE.

Reply:
This is mostly a spec verbiage. I found it easier to reuse the verbiage of the MSE.

readyState of the video element is indeed unchanged with MMS. But it does drive on when the streaming event will be fired

Reply:
Probably this text should just refer to the streaming event then rather than readyState, but defer to spec editors.

Reply:
done, reworked it all.